roles/openshift_logging_kibana/templates/kibana.j2

```diff
@@ -143,6 +143,30 @@ spec:
                resourceFieldRef:
                  containerName: kibana-proxy
                  resource: limits.memory
+{% if (openshift_http_proxy is defined and openshift_http_proxy is not none and openshift_http_proxy != "") %}
+            -
+             name: "http_proxy"
```

Comment:
is there a reason we have this env var in both cases?

Reply:
I usually set the proxy vars in both cases as some tools (can't remember which one) can't detect the proxy if the variables are not set with the appropriate case.

For all tools linked with libcurl, uppercase http_proxy is not checked for security reasons (https://github.com/curl/curl/blob/master/lib/url.c#L2258). So if we only have tools linked with libcurl in the container image, than I agree HTTP_PROXY variable is useless. Shall I keep only lowercase definition for all three env variables?